ios/FluentUI.Demo/FluentUI.Demo/Info.plist

```diff
@@ -26,7 +26,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>130</string>
+	<string>131</string>
```

Comment:
I think we have to continue the numbering based off of the 0.8 branch, right?

Reply:
https://github.com/microsoft/fluentui-apple/commit/2939483015fc9c08a74ca9e1edafa6e6f33c2682 also do we want to adopt a .x numbering system for the other string values too? We used to not do that I think

Reply:
good point. we can't decrement numbers once you publish it. but i think once in for all we should just be x.x.x in all of our info.plist to avoid this type of mistakes.